crates/subspace-networking/src/node_runner.rs

```diff
@@ -584,16 +584,23 @@ where
                     kademlia.add_address(&peer_id, address);
                 }
             } else {
-                trace!(
+                info!(
```

Comment:
I don't think this should be `info`, left from debugging?

Reply:
Hmm, I wanted to set to `warn` initially similar to multiple types of `incorrect block` messages from our node. Ok, I'll set it to debug.